docs/source/io_formats/settings.rst

```diff
@@ -449,24 +449,29 @@ attributes/sub-elements:
 
     *Default*: 1.0
 
+  :type:
+    Indicator of source type. One of ``independent_source``, ``file``, or ``compiled``.
```

Comment:
The files below only have `type='independent'` - either this documentation or those files should change...

Reply:
The other classes use the unique component of the class name as their type, so we should do the same here. Updating that docstring now.